libraries/AP_Airspeed/AP_Airspeed_I2C.cpp

```diff
@@ -80,7 +80,7 @@ void AP_Airspeed_I2C::_collect()
 
     _measurement_started_ms = 0;
 
-    if (_dev->transfer(nullptr, 0, data, sizeof(data))) {
+    if (!_dev->read_registers(0, data, sizeof(data))) {
```

Comment:
A bit more on this:

Apparently we have 2 different devices: one in "standard mode" and a variant "-L" designed for low power.  The first does continuous measurements, so there's no reason to send the smbus_quick(). The second needs the smbus_quick() to wake up and do a new measurement.

See "Low Power Option": http://www.mouser.com/ds/2/261/MS4525DO-472989.pdf
And the flow diagrams: http://www.meas-spec.com/downloads/Configuration,_POR_and_Power_Consumption.pdf